client/src/style/button/button.pcss

```diff
@@ -0,0 +1,7 @@
+.button {
```

Comment:
Our convention in consulting is to use colocation whenever possible -- i.e., grouping files by functionality rather than by type. So, 

client/src/components/homes/Home.tsx
client/src/components/homes/styles/home.pcss

For something general like button.pcss, I recommend

client/src/components/common/styles/button.pcss